src/rust/ensogl/src/math/geometry/triangle.rs

```diff
@@ -0,0 +1,33 @@
+use core::f32::consts::PI;
+use crate::display::shape::Var;
+use crate::math::algebra::{Sin, Cos, Sqrt};
+use crate::math::topology::unit::{Radians, Angle};
+use std::ops::*;
+
+pub struct Triangle<S> {
```

Comment:
Plus, where this sturcture is used? I don't see any usages now.

Reply:
This might be used when using `Plane().cut_angle` instead of `Triangle`. I've removed it for now and will add a clean version if it is needed again.